setup.py

```diff
@@ -210,6 +212,9 @@ def compile_native_invocation_handler(*possible_homes):
     if PLATFORM == 'win32':
         INCL_DIR = join(JDK_HOME, 'include', 'win32')
         LIBRARIES = ['jvm']
+    elif PLATFORM == 'sunos5':
+        INCL_DIR = join(jdk_home, 'include', 'solaris')
```

Comment:
I don't see where the lower-case `jdk_home` is defined? A linter would have caught that :cry:

Reply:
https://github.com/kivy/pyjnius/pull/446